src/main/kotlin/odyseja/odysejapka/gad/Team.kt

```diff
@@ -50,4 +54,11 @@ data class Team(
             zspSheet = zspSheet
         )
     }
+    fun getCityFromTeamName(teamName: String):String{
```

Comment:
Wystarczy `getCityName` to, że to jest z brane z team name to juz szczegół implementacyjny